src/mirco_inputparameters.cpp

```diff
@@ -9,8 +9,9 @@ namespace
   // Shape factors (See section 3.3 of https://doi.org/10.1007/s00466-019-01791-3)
   // These are the shape factors to calculate the elastic compliance correction of the micro-scale
   // contact constitutive law for various resolutions.
-  // NOTE: Currently MIRCO works for resoulution of 1 to 8. The following map store the shape
-  // factors for resolution of 1 to 8.
+  // NOTE: MIRCO works most accurately the integer resolutions of 1 to 8, which are given below. For
+  // other values of the (equivalent) resolution, shape factors are interpolated/extrapolated
+  // linearly.
```

Comment:
@PhilipOesterlePekrun This is at the core of this PR! These numbers for n= 1,...,8 come from simulations with a flat indenter (see Bonari et al. (2020)). I think it makes sense to just compute them for n=9 (or even higher, if possible), so that we do not need to rely on extrapolation.

Essentially, I propose to compute those numbers and add them to the `const std::map<int, double>` shape factors.

Another thought on testing: so far, we just assume that these numbers are correct and we use them without further testing. It makes sense to use a pre-computed value in actual computations, though we could verify it "once in a while", i.e. nightly. I am wondering whether one can setup a series of tests, that actually compute these values and compare them against the hard-coded values in MIRCO.

Reply:
@PhilipOesterlePekrun, let's add some shape factors for n=9 and n=10 for both pressure and point forces. That should not be too hard. 

Feel free to open an issue about possible automation and testing of these factors, just that we do not forget about it. I do not prioritize this topic very much at the moment. We should discuss this with @RShaw026 @jacopenji.